docs/UserGuide.md

```diff
@@ -131,6 +131,54 @@ In this guide, the syntax / format of a command is shown like this:
 6. Refer to the [Features](#6-command-summary) below to see which commands TutorAid understands.
 <hr>
 
+# 4. Beginner's tutorial
+
+Welcome to TutorAid's beginner's tutorial! Here, you will learn how you can use some of our commands to manage the details of your students and lessons on a day-to-day basis.
+
+If this is your first time launching TutorAid, you will be able to see that some sample data has been provided to you. For this tutorial, let's assume that you tutor the students
+shown in the student panel and that you offer the lesson shown in the lesson panel.
+
+### Step 4: Adding a progress note for a student
+
+Now, imagine you have ended your `Maths 1` lesson. You realised that one of your students, `Alex Yeoh`, seems to struggle with the topic `Vectors`, and you wish to take
+note of it so that you can provide him extra practice. To do so, you can easily add a progress note for `Alex Yeoh ` in TutorAid by following the steps below:
```

Comment:
Minor nitpick here, but the ending space character in `Alex Yeoh ` should be removed, i.e. from `Alex Yeoh `  to `Alex Yeoh`